api/server/credentials_linux.go

```diff
@@ -213,13 +215,8 @@ func (s *Server) LogAction(w http.ResponseWriter, r *http.Request) error {
 			break
 		}
 		message = fmt.Sprintf("LoginUID=%v, %s", loginuid, message)
-		if uint32(loginuid) == 0xffffffff { // -1 means no login user
-			//No login UID is set, so no point in looking up a name
-			break
-		}
-
```

Comment:
I'd like to keep a range check here, breaking out if loginuid is out of range for an unsigned 32-bit, before we cast it to a uint32 for passing to getpwuid().